### ISSUE TITLE
geOps-API geht nicht mehr

### ISSUE THREAD

**Issue by Shure79** [florianlederer/mvv-display-for-ESP32#34]
Die Beispiele in der config.h.dist funktionieren, solange die mvg_api benutzt wird (Marienplatz). Das Beispiel mit der geops_api (Hauptbahnhof) klappt leider nicht.

**Comment by Shure79** [comment]
Ich hab es noch mit dem derzeitig gültigen Demo-API-Key von geOps probiert, allerdings ohne Erfolg.

**Comment by florianlederer** [comment]
Ich habe leider gerade keine Hardware bei mir. Der Key und die Stationsnummer sollten noch passen. Auf meinem Computer kann ich ganz normal mit dem Server kommunizieren. 

Hast du mal einen anderen Bahnhof probiert? Ist die Wifi Verbindung korrekt für die zweite Konfiguration eingetragen?

**Comment by Shure79** [comment]
Ja, die Kommunikation mit dem Server über einen WebSocket Client klappt. Ich hab das grade gegengecheckt. Ein anderer Bahnhof geht ebenfalls nicht und die WiFi-Verbindung steht (sehe ich auch im Router) und es ist auch nur eine Konfiguration eingetragen.
Die serielle Konsole gibt folgendes aus:
Try to Reconnect geops!
Not Connected!

Wenn ich das richtig interpretiere liegt der Fehler irgendwo in "void init_geops_api(Config config)".
